### DISCUSSION TITLE
EAGE-HACK-2024: I need a team!

### DISCUSSION THREAD

**Discussion by aruss175** [EAGE-Annual-Hackathon/EAGE-Annual-Hackathon-Discussion-Boards#12] [General]
Use this thread if you need a team, or you have a team looking for some more members!

**Comment by ovcharenkoo** [comment]
**To register please select the "Hackathon" option during registration for the EAGE Annual event**

**Comment by ValentinCassayre** [comment]
Hey,
I guess I am the first one... A bit about myself : I'm 22 years old and come from France, currently studying geophysics with specialization in machine learning. I'm eager to participate in my first EAGE Annual hackathon after assisting to the excellent presentations last year in Vienna. I'm looking forward to joining or creating a team, I don't have any specific requirements, neither subject in mind. 
Feel free to message me,
Valentin

**Reply by Godor9** [reply]
Thanks for the post!  I was curious why there is not many post!. Maybe we can make a team then

**Reply by giuseppe-amendola** [reply]
Hi, I am also keen to team up! I'm Geophysicist with some data science background and passionate about ML.
Let's keep in touch,
Giuseppe

**Reply by Godor9** [reply]
Hi, guys I didn't receive any info about the online session. I don't know what is going on!
recently I did some things with Nvidia modulus to do ML .  It is about solving PDE with neural operator. 
contact me through: hongliang.wang@ife.no

**Reply by aruss175** [reply]
We are in person here in Oslo in Room 8 at the EAGE conference!  This hackathon is in person only.
